docs/v12/documentation/accessibility.md

```diff
@@ -66,5 +64,5 @@ When we publish new content, we’ll continue to make sure that it meets accessi
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 5 November 2021. It was last reviewed on 6 September 2022.
+This statement was prepared on 5 November 2021. It was last reviewed on 11 January 2024.
```

Comment:
I'm not 100% sure we should update this date – although we're revising the statement to update the contact details, we haven't _reviewed_ it (i.e. checked that it's still an accurate representation of the accessibility of the docs site) in any way.

@selfthinker do you have any thoughts on what we should do here?

Reply:
@36degrees Anika may not be able to review this as she's focused on GOV.UK audit work currently. I agree with your note though so let's revert it back to the original date.